README.md

```diff
@@ -90,6 +90,24 @@ npm run mem_backend &
 npm run ft_test
 ```
 
+## Configuring
+
+If you want to specify a specific endpoint (different from localhost),
+you need to add it to your config.json:
+
+```shell
+"regions": {
+
+     ...
+     "localregion": ["localhost"],
+     "specifiedregion": ["myhostname.com"]
+},
+```
+
+Our S3server support both:
+path-style: http://myhostname.com/mybucket
+hosted-style: http://mybucket.myhostname.com
+
```

Comment:
However, note that hosted-style requests will not hit the server if you are using an ip address for your host.  So, make sure you are using path-style requests in that case.  For instance, if you are using the AWS SDK for JavaScript you would instantiate your client like this:

``javascript
const s3 = new aws.S3({
    endpoint: 'http://localhost:8000',
   s3ForcePathStyle: true,
 });
``